packages/origin-ui-core/src/components/Organization/OrganizationView.tsx

```diff
@@ -70,23 +70,28 @@ export function OrganizationView() {
 
     useEffect(() => {
         const getOrganization = async () => {
-            const iRecOrg = await iRecClient.getRegistrations();
+            let iRecOrg;
+            if (enabledFeatures.includes(OriginFeature.IRec)) {
+                iRecOrg = await iRecClient.getRegistrations();
+            }
             const organization = params.id
                 ? await organizationClient.getById(parseInt(params.id, 10))
                 : userOffchain?.organization;
             if (organization) {
                 setValues(organization);
 
-                if (params.id) {
-                    setIRecData(
-                        iRecOrg?.filter(
-                            (org) => parseInt(org.owner, 10) === parseInt(params.id, 10)
-                        )[0]
-                    );
-                } else {
-                    setIRecData(
-                        iRecOrg?.filter((org) => parseInt(org.owner, 10) === organization.id)[0]
-                    );
+                if (enabledFeatures.includes(OriginFeature.IRec)) {
+                    if (params.id) {
+                        setIRecData(
+                            iRecOrg?.filter(
+                                (org) => parseInt(org.owner, 10) === parseInt(params.id, 10)
+                            )[0]
+                        );
+                    } else {
+                        setIRecData(
+                            iRecOrg?.filter((org) => parseInt(org.owner, 10) === organization.id)[0]
+                        );
+                    }
```

Comment:
Instead of 

```javascript
                    if (params.id) {
                        setIRecData(
                            iRecOrg?.filter(
                                (org) => parseInt(org.owner, 10) === parseInt(params.id, 10)
                            )[0]
                        );
                    } else {
                        setIRecData(
                            iRecOrg?.filter((org) => parseInt(org.owner, 10) === organization.id)[0]
                        );
                    }
```
you can simply write
```javascript

const owner = params.id ? parseInt(params.id, 10) : organization.id;
setIRecData(
    iRecOrg?.filter((org) => parseInt(org.owner, 10) === owner)[0]
);
```